tests/csm/rest/test_capacity_quota.py

```diff
@@ -0,0 +1,442 @@
+# pylint: disable=too-many-lines
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""Tests System capacity scenarios using REST API
+"""
+import logging
+import math
+import time
+from http import HTTPStatus
+from random import SystemRandom
+from time import perf_counter_ns
+
+import pytest
+
+from commons import configmanager
+from commons import cortxlogging
+from commons.utils import assert_utils
+from libs.csm.csm_interface import csm_api_factory
+from libs.csm.rest.csm_rest_quota import GetSetQuota
+from libs.s3 import s3_misc, s3_test_lib
+
+# pylint: disable=too-many-instance-attributes
+class TestSystemCapacity():
```

Comment:
Duplicate class name. We need unique name for each test class. It should be added to csm_interface.py